src/components/login/constants.ts

```diff
@@ -0,0 +1,50 @@
+/**
+ * Authorization config types for SSO
+ */
+export const AUTHORIZATION_CONFIG_TYPES = {
```

Comment:
Please rename all the const in UPPER_CASE convention

Reply:
will handle in the active/inactive story